.env.template

```diff
@@ -9,6 +9,8 @@ REACT_APP_INFURA_PROJECT_ID=464bfe80298f4d0782f61h4b91de2dcb
 #
 # true, false
 REACT_APP_IS_MAINNET=true
+# string (only if REACT_APP_IS_MAINNET=false)
```

Comment:
Just to be clear, even if `REACT_APP_IS_MAINNET=true`, it could be set to an alternative testnet name (not the default string Prymont).